projects/etos_suite_runner/src/etos_suite_runner/esr.py

```diff
@@ -122,7 +177,12 @@ def _request_environment(self, ids: list[str], otel_context_carrier: dict) -> No
         otel_context = TraceContextTextMapPropagator().extract(carrier=otel_context_carrier)
         otel_context_token = opentelemetry.context.attach(otel_context)
         try:
-            self.__request_environment(ids)
+            # TestRun identifier, correlates to the TERCC that should be sent when
```

Comment:
What do we mean when we say correlate? Is this a conditional on whether to run with Controller or not?

Reply:
Currently it means that the TestRun identifier is the TERCC event ID, but that might change in the future